Assets/Scripts/App.cs

```diff
@@ -1919,12 +1923,16 @@ public enum AppState {
       Application.OpenURL(url);
     }
 #else
-    // Something about the url makes OpenURL() not work on OSX, so use a workaround
-    if (Application.platform == RuntimePlatform.OSXEditor ||
-        Application.platform == RuntimePlatform.OSXPlayer) {
-      System.Diagnostics.Process.Start(url);
-    } else {
-      Application.OpenURL(url);
+    // Something about the url makes OpenURL() not work on OSX or Linux, so use a workaround
```

Comment:
This comment doesn't match the update; did you mean to add the Linux RuntimePlatforms to the cases?